subgraph/subgraph.template.yaml

```diff
@@ -0,0 +1,71 @@
+specVersion: 1.0.0
+description: Subgraph Indexer for MACI contract
+repository: https://github.com/hangleang/maci-subgraph
+indexerHints:
+  prune: auto
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum
+    name: MACI
+    network: { { network } }
```

Comment:
```suggestion
    network: {{ network }}
```

Reply:
this file will be removed! use the `--network` flag when deploy the subgraph will generate `subgraph.yaml` with corresponding network config from `networks.json` file